index.html

```diff
@@ -19,6 +19,23 @@
 <link rel="stylesheet" type="text/css" href="css/menu.css" />
 <link rel="stylesheet" type="text/css" href="lib/Font-Awesome/css/font-awesome.min.css" />
 </head>
+
+<script>
+  /* 
+   * This code is to test if fat arrows are supported.
+   * If not, an error message is displayed to the user suggesting him to change its current browser.
+   * It is placed here to avoid the initial error parsing which is done when it's placed in a JS file.
+   */ 
```

Comment:
````js
/*
 * Test if fat arrows are supported.
 *
 * We do this here as browsers have "interesting" ways of parsing code in `eval` in js files, which causes the entire file to fail parsing instead of just the offending string.
 */ 
````